.github/workflows/bump-nocache-line.yml

```diff
@@ -16,19 +16,20 @@ jobs:
     runs-on: ubuntu-latest
     container: registry.fedoraproject.org/fedora:latest
     steps:
+      - name: Install dependencies
+        run: dnf install -y git-core
       - name: Check out repository
         uses: actions/checkout@v2
         with:
           ref: main
-          fetch-depth: 0
```

Comment:
I believe this is necessary because of `push-to-fork`.  Otherwise if the forked repo is behind and we don't have the history to bring it up to date, the push could fail.

Reply:
thanks - fixed